indico/modules/events/papers/settings.py

```diff
@@ -46,6 +46,8 @@ def to_python(value):
     'judge_deadline': None,
     'layout_reviewer_deadline': None,
     'content_reviewer_deadline': None,
+    'scale_lower': 0,
```

Comment:
Let's discuss that next year 😛 
But yeah, for now I'll leave it at -3...+3 like legacy.